### PR TITLE
Fixed some emoji descriptions for better accessibility

### PR DESCRIPTION
**Pull request by dominikwilkowski**
Finding them all isn't easy so I thought I quickly create PR if I find some and so I have :)

### CODE REVIEW

File: docs/pages/releases/2021-06-02.mdx [hunk 1]
```diff
@@ -41,7 +41,7 @@ In addition to the JSON one above, we added new examples for:
 - Setting [Default Values](https://github.com/keystonejs/keystone/tree/main/examples/default-values) for fields.
 - [Extending the GraphQL Schema](https://github.com/keystonejs/keystone/tree/main/examples/extend-graphql-schema) with custom queries and mutations.
 
-We also published a tutorial that shows you [how to embed Keystone and SQLite in a Next.js app](https://keystonejs.com/tutorials/embedded-mode-with-sqlite-nextjs). The end result is an app with a queryable GraphQL endpoint based on your Keystone schema that you can run live on Vercel – for free! <Emoji symbol="🚀" alt="Rocket"/>
+We also published a tutorial that shows you [how to embed Keystone and SQLite in a Next.js app](https://keystonejs.com/tutorials/embedded-mode-with-sqlite-nextjs). The end result is an app with a queryable GraphQL endpoint based on your Keystone schema that you can run live on Vercel – for free! <Emoji symbol="🚀" alt="Rocket"/>
```
Comment (dominikwilkowski) [inline docs/pages/releases/2021-06-02.mdx:44]: This one as an interesting one. Found a non-standard empty space and removed it

File: docs/pages/releases/2021-06-02.mdx [hunk 2]
```diff
@@ -175,7 +175,7 @@ postsCount(where: PostWhereInput! = {}): Int
 
 We've updated our Prisma dependency from `2.22.1` to `2.24.0`! Check out the [Prisma release notes](https://github.com/prisma/prisma/releases) for more details.
 
-### Credits <Emoji symbol="Party popper"/>
+### Credits <Emoji symbol="🎉" alt="Party popper"/>
```
Comment (dominikwilkowski) [inline docs/pages/releases/2021-06-02.mdx:178]: This one didn't gain attention as it looked almost normal but was missing it's emoji

Comment (bladey) [reply docs/pages/releases/2021-06-02.mdx:178]: Good catch!

File: docs/pages/releases/2021-06-02.mdx (hunk 1 above)
Comment (bladey) [reply docs/pages/releases/2021-06-02.mdx:44]: Thanks!

**Review by bladey — APPROVED** [review]
Very helpful, thanks @dominikwilkowski.

**Pull request merged**